docs/triage-tool.md

```diff
@@ -291,6 +291,24 @@ but that if JAX is moved forward to include
 then the test fails.
 This failure is fixed in [jax#24427](https://github.com/jax-ml/jax/pull/24427).
 
+## Triaging versioned components
+In addition to the git-repository-based triage discussed above, the tool is able to
+include a limited set of versions of other software components in the multi-component
+triage algorithm.
+To enable this feature, `--build-scripts-path` must give a path **inside** the test
+environment (container) that contains executables named `installCOMPONENT.sh` that
+accept a version as the first argument.
+This is typically used in conjunction with `--container-mount` to allow those
+installation helpers to be injected from the host system.
+
+Versions of `COMPONENT` will be read from the last-known-good and first-known-bad
+container environment variable `COMPONENT_VERSION`, or the version of `COMPONENT` can
+be set explicitly via `--{passing,failing}-commits`.
+
+Only two versions of `COMPONENT` will be read, one from each end of the triage range.
+If those versions are the same as each other, `installCOMPONENT.sh` will not be called
+and need not exist.
```

Comment:
What will happen if a component changes, but the script don't exist? Is it useful to mention it?

Reply:
It was attempt to avoid enumerating the installation scripts and keep this implicit, but I should probably just explicitly list the available scripts at both endpoints and use that for an explicit check.

Reply:
This is now explicit:
```
                logger.warning(
                    f"No installation scripts found for: {packages_missing_scripts}, "
                    "whose versions change across the bisection range. These will be "
                    "excluded from the bisection, which may cause it not to converge!"
                )
```